Kubernetes/README.md

```diff
@@ -1,34 +1,38 @@
-# Vagrantfile to setup a Kubernetes Cluster on Oracle Linux 7
-This Vagrantfile will provision a Kubernetes cluster with one master and _n_
+# Vagrant project to setup a Kubernetes Cluster on Oracle Linux 7
+
+This Vagrant project will provision a Kubernetes cluster with one master and _n_
```

Comment:
Recommend we stick with referring to "master" nodes until Kubernetes upstream changes this terminology.